indico/htdocs/sass/partials/_icons.scss

```diff
@@ -154,6 +154,9 @@ i[class^="icon-"]:before {
 .icon-edit:before {
     content: "\2e";
 }
+.icon-enlarge7:before {
```

Comment:
Just noticed that rather awful name. Can we change this in the icomoon app so we can use a decent name without having to manually rename it everytime we add a new Icon? ping @OmeGak